templates/archive-list.html

```diff
@@ -1,16 +1,14 @@
 <ul class="p-list">
-    <li class="p-list__item"><a href="/archives/2018/01">January 2018</a></li>
-    <li class="p-list__item"><a href="/archives/2017/12">December 2017</a></li>
-    <li class="p-list__item"><a href="/archives/2017/11">November 2017</a></li>
-    <li class="p-list__item"><a href="/archives/2017/10">October 2017</a></li>
-    <li class="p-list__item"><a href="/archives/2017/09">September 2017</a></li>
-    <li class="p-list__item"><a href="/archives/2017/08">August 2017</a></li>
-    <li class="p-list__item"><a href="/archives/2017/07">July 2017</a></li>
-    <li class="p-list__item"><a href="/archives/2018/06">June 2017</a></li>
-    <li class="p-list__item"><a href="/archives/2017/05">May 2017</a></li>
-    <li class="p-list__item"><a href="/archives/2017/04">April 2017</a></li>
-    <li class="p-list__item"><a href="/archives/2017/03">March 2017</a></li>
-    <li class="p-list__item"><a href="/archives/2017/02">February 2017</a></li>
-    <li class="p-list__item"><a href="/archives/2017/01">January 2017</a></li>
-    <li class="p-list__item"><a href="/archives/2016/12">December 2016</a></li>
-  </ul>
\ No newline at end of file
+  {% for year in range(today.year,2006,-1) %}
+  <li class="p-list__item">{{ year }}
+    <ul class="p-inline-list--middot">
+    {% for month in range(1, 13, 1) %}
+      {% if today.year == year and month > today.month %}
```

Comment:
Could this not be `{% if today.year != year or month < today.month %}` and then remove the `{% else %}`?

Reply:
@Lukewh not really... it is trying to NOT print Feb 2018 as it hasn't happened... I think your's isn't the inverse  - however this is - {% if today.year != year or (today.year == year and month <= today.month) %} but more complicated

Reply:
ah, yeah you're right. I prefer the more complicated approach but I'm okay with you not changing it if you don't want to.

Reply:
fixed